core/app/modules/kit-library/assets/js/data/component.js

```diff
@@ -1,11 +1,14 @@
-import * as dataCommands from './commands';
+import { Index, KitsDownloadLink } from './commands';
 
 export default class Component extends $e.modules.ComponentBase {
 	getNamespace() {
 		return 'kits';
 	}
 
 	defaultData() {
-		return this.importCommands( dataCommands );
+		return this.importCommands( {
+			index: Index,
+			downloadLink: KitsDownloadLink,
+		} );
```

Comment:
```suggestion
		return this.importCommands( commandsData );
```

Reply:
The name of the class and the name of the key of the object are different.

Reply:
so change the class name to 'DownloadLink' same with the endpoint it's already in the kit.